src/devices/awox.ts

```diff
@@ -65,6 +65,20 @@ export const definitions: DefinitionWithExtend[] = [
         description: "Dimmable filament lamp",
         extend: [m.light()],
     },
+    {
+        zigbeeModel: ["EZMB-RGB-TW-CLB"],
+        model: "EZMB-RGB-TW-CLB",
+        vendor: "AwoX",
+        description: "EGLO Zigbee RGB+TW Ceiling light",
+        extend: [
+            m.deviceEndpoints({endpoints: {"1": 1, "3": 3}}),
```

Comment:
I think this can be removed (please check if everything works fine without). Same goes for `meta: {multiEndpoint: true},`